contracts/src/interfaces/ITeleporterTokenDestination.sol

```diff
@@ -13,6 +13,24 @@ import {TeleporterFeeInfo} from "@teleporter/ITeleporterMessenger.sol";
  * DO NOT USE THIS CODE IN PRODUCTION.
  */
 
+/**
+ * @notice Settings for constructing a {ITeleporterTokenDestination} contract.
+ * @param teleporterRegistryAddress the current blockchain id's Teleporter registry
+ * address. https://github.com/ava-labs/teleporter/tree/main/contracts/src/Teleporter/upgrades.
+ * @param teleporterManager address that manages this contract's integration with the
+ * Teleporter registry and Teleporter versions.
+ * @param sourceBlockchainID The blockchain ID of the associated source token bridge
+ * @param tokenSourceAddress The address of the source token bridge contract.
+ * @param tokenSymbol The symbol for the token of this destination bridge contract.
+ */
+struct TeleporterTokenDestinationSettings {
+    address teleporterRegistryAddress;
+    address teleporterManager;
+    bytes32 sourceBlockchainID;
+    address tokenSourceAddress;
+    string tokenSymbol;
```

Comment:
I think this is a little cleaner if we don't include `tokenSymbol` in here. Then we can make the `TeleporterTokenDestination` constructor take in the whole `TeleporterTokenDestinationSettings` object.